app/services/user.js

```diff
@@ -18,3 +18,13 @@ exports.createUser = async user => {
     throw databaseError(error);
   }
 };
+
+exports.findUserBy = async (value,attribute = 'email') => {
```

Comment:
This is fine, but as suggestion: you can just receive an interface (object)
```suggestion
exports.findUserBy = async params => {
```

and execute it like `findUserBy({email})`. Either way is fine